### DISCUSSION TITLE
Welcome to huggingfaceR Discussions!

### DISCUSSION THREAD

**Discussion by farach** [farach/huggingfaceR#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OscarKjell** [comment]
Hi everyone, 
I'm excited to see your work in this space. Have you seen our R-package [text](http://r-text.org/), which provide the user access to huggingface models? 
The package is also on CRAN (https://cloud.r-project.org/web/packages/text/index.html), and has a [tutorial preprint](https://psyarxiv.com/293kt/) with social scientists as a target audience. Text currently provides word embeddings from transformers models, as well as some high level functions to, for example, train and visualise words in the word embedding space.

**Reply by jpcompartir** [reply]
Hi Oscar, 

Can't speak for Sam & Alex, but can't say that I had seen it myself 👍
